test/Microsoft.Dnx.DesignTimeHost.FunctionalTests/Infrastructure/DthTestServer.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Threading;
+using Microsoft.Dnx.CommonTestUtils;
+using Microsoft.Dnx.Runtime;
+
+namespace Microsoft.Dnx.DesignTimeHost.FunctionalTests.Infrastructure
+{
+    public class DthTestServer : IDisposable
+    {
+        private static readonly string DthName = "Microsoft.Dnx.DesignTimeHost";
```

Comment:
Private fields -> Constant; public fields or protected in public classes-> Readonly